package.json

```diff
@@ -148,6 +148,12 @@
     ]
   },
   "dependencies": {
+    "@react-aria/checkbox": "^3.2.1",
```

Comment:
Remove direct dependency on @react-aria. We already have it in @react-native-aria/checkbox